lib/WPA/Andersen.cpp

```diff
@@ -187,6 +190,18 @@ void Andersen::initialize()
  */
 void Andersen::finalize()
 {
+    // TODO: check -stat too.
```

Comment:
What's these few lines for in the finalize method of Andersen?

Reply:
This is to print the statistics of how effective the clustering actually was (how many words needed).